packages/openactive-integration-tests/test/report-generator.js

```diff
@@ -18,6 +19,12 @@ class ReportGenerator {
       return chalkFn(options.fn(this));
     });
 
+    Handlebars.registerHelper("renderSuiteName", function(suiteName, options) {
+      if (suiteName.length <= 2) return "Test setup";
+
+      return suiteName.slice(2).join(" >> ");
+    });
```

Comment:
I can't remember the output from before. Are the first two parts of the suite name just not useful, common phrases?

Reply:
the first two parts of suite name are already referenced by the filename, and were just extra noise